manifest.json

```diff
@@ -1,82 +1,82 @@
   },
   {
     "theme": "escapade",
-    "new_version": "1.1.0",
+    "new_version": "1.1.1",
     "url": "https://wpnux.godaddy.com/?theme=escapade",
-    "package": "https://github.com/godaddy/wp-escapade-theme/releases/download/v1.1.0/escapade.zip",
-    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-escapade-theme/v1.1.0/screenshot.png",
+    "package": "https://github.com/godaddy/wp-escapade-theme/releases/download/v1.1.1/escapade.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-escapade-theme/v1.1.1/screenshot.png",
     "name": "Escapade"
   },
   {
     "theme": "lyrical",
-    "new_version": "1.1.1",
+    "new_version": "1.1.2",
     "url": "https://wpnux.godaddy.com/?theme=lyrical",
-    "package": "https://github.com/godaddy/wp-lyrical-theme/releases/download/v1.1.1/lyrical.zip",
-    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-lyrical-theme/v1.1.1/screenshot.png",
+    "package": "https://github.com/godaddy/wp-lyrical-theme/releases/download/v1.1.2/lyrical.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-lyrical-theme/v1.1.2/screenshot.png",
     "name": "Lyrical"
   },
   {
     "theme": "mins",
-    "new_version": "1.1.1",
+    "new_version": "1.1.2",
     "url": "https://wpnux.godaddy.com/?theme=mins",
-    "package": "https://github.com/godaddy/wp-mins-theme/releases/download/v1.1.1/mins.zip",
-    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-mins-theme/v1.1.1/screenshot.png",
+    "package": "https://github.com/godaddy/wp-mins-theme/releases/download/v1.1.2/mins.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-mins-theme/v1.1.2/screenshot.png",
     "name": "Mins"
   },
   {
     "theme": "primer",
-    "new_version": "1.8.2",
+    "new_version": "1.8.3",
     "url": "https://wpnux.godaddy.com/?theme=primer",
-    "package": "https://github.com/godaddy/wp-primer-theme/releases/download/v1.8.2/primer.zip",
-    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-primer-theme/v1.8.2/screenshot.png",
+    "package": "https://github.com/godaddy/wp-primer-theme/releases/download/v1.8.3/primer.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-primer-theme/v1.8.3/screenshot.png",
     "name": "Primer"
   },
   {
     "theme": "scribbles",
-    "new_version": "1.1.0",
+    "new_version": "1.1.1",
     "url": "https://wpnux.godaddy.com/?theme=scribbles",
-    "package": "https://github.com/godaddy/wp-scribbles-theme/releases/download/v1.1.0/scribbles.zip",
-    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-scribbles-theme/v1.1.0/screenshot.png",
+    "package": "https://github.com/godaddy/wp-scribbles-theme/releases/download/v1.1.1/scribbles.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-scribbles-theme/v1.1.1/screenshot.png",
     "name": "Scribbles"
   },
   {
     "theme": "stout",
-    "new_version": "1.1.0",
+    "new_version": "1.1.3",
```

Comment:
The jump from 1.1.0 to 1.1.3 was due to the increments that were required when getting this uploaded to WordPress.org. I guess we didn't update the manifest file after updated Stout.